global_install/hooks/.pre-commit-config.yaml

```diff
@@ -10,4 +10,9 @@ repos:
     hooks:
     -   id: detect-secrets
         args: ['--baseline', '.secrets.baseline']
-        exclude: .*/tests/.*
+        exclude: .*/tests/.*|
```

Comment:
I don't think we should blanket exclude tests. Test data is a prima the facia case where real secrets might be committed.

Reply:
That's a really good point. Nice catch!